test/test_runner.sh

```diff
@@ -27,9 +27,13 @@ trap drop_db EXIT
 cd sql/
 # install lanterndb extension
 psql "$@" -U ${DB_USER} -d postgres -v ECHO=none -q -c "DROP DATABASE IF EXISTS ${TEST_CASE_DB};" 2>/dev/null
-psql "$@" -U ${DB_USER} -d postgres -v ECHO=none -q -c "DROP DATABASE IF EXISTS ${TEST_CASE_DB};" 2>/dev/null
 psql "$@" -U ${DB_USER} -d postgres -v ECHO=none -q -c "CREATE DATABASE ${TEST_CASE_DB};" 2>/dev/null
-psql "$@" -U ${DB_USER} -d ${TEST_CASE_DB} -v ECHO=none -q -c "SET client_min_messages=error; CREATE EXTENSION vector; CREATE EXTENSION lanterndb;" 2>/dev/null
+VECTOR_AVAILABLE=$(psql -U ${DB_USER} -t -c "SELECT 1 FROM pg_available_extensions WHERE name = 'vector';" -tA)
+if [ "$VECTOR_AVAILABLE" = "1" ]; then
+    psql "$@" -U ${DB_USER} -d ${TEST_CASE_DB} -v ECHO=none -q -c "SET client_min_messages=error; CREATE EXTENSION vector; CREATE EXTENSION lanterndb;" 2>/dev/null
```

Comment:
I think we should avoid creating `vector` extension here and should instead assume `sql` files requiring that extension will have a `CREATE EXTENSION vector;` line.
That way, we can make sure that the rest of the tests run without enabling the vector extension.
As is, it would be really easy to have a hidden dependency on vector from other test files without realizing it since we have pgvector installed in ci/cd and on our dev machines.

Reply:
I'm happier with that.